packages/react-devtools/app.html

```diff
@@ -150,13 +162,13 @@
 
       const $localhost = $("#localhost");
       $localhost.innerText = `<script src="http://localhost:${port}"></` + 'script>';
-      $localhost.addEventListener('click', selectAll);
-      $localhost.addEventListener('focus', selectAll);
+      $localhost.addEventListener('click', selectAllAndCopy);
+      $localhost.addEventListener('focus', selectAllAndCopy);
```

Comment:
Huh, I just noticed that this has the unexpected effect of re-focusing (and re-copying) every time you tab away from and back to the process/window. Probably fine though. Overall this seems like a nice improvement.